### DISCUSSION TITLE
Property "suggest.autoTrigger": "none" is not working anymore.

### DISCUSSION THREAD

**Discussion by amgodoi** [neoclide/coc.nvim#3767] [Q&A] [answered]
Hi!

I just updated COC to:
coc.nvim version: 0.0.80-d02edd6b 2022-04-17 20:25:31 +0800

After that  "suggest.autoTrigger": "none" is not working anymore. After update it is always triggered ... I tested it using java and python.

Some bug introduced?

**Comment by chemzqm** [comment, marked as answer]
Fixed on latest release branch of coc.nvim.

**Reply by amgodoi** [reply]
Worked!!!
Thanks a lot.
